libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.kt

```diff
@@ -30,9 +30,9 @@ class RouteUtils {
      * @return true if in arrival state, false if not
      */
     fun isArrivalEvent(routeProgress: RouteProgress): Boolean =
-        routeProgress.currentState()?.let { currentState ->
-            currentState == RouteProgressState.ROUTE_ARRIVED
-        } ?: false
+            routeProgress.currentState()?.let { currentState ->
```

Comment:
Changes here (`RouteUtils`) are only formatting. Do we need them at all?

Reply:
@olegzil ptal at `ktlint-setup-and-using.md` and apply ktlint rules to this project. If it was done, than we could keep this changes because it's our rules by default imo :)